error.go

```diff
@@ -0,0 +1,27 @@
+package microstorage
+
+import (
+	"github.com/juju/errgo"
+)
+
+var InvalidConfigError = errgo.New("invalid config")
```

Comment:
Random brain wave. When we have the `New` function in `microerror` we can fully abstract `errgo` and completely get rid of the `errgo` dependency in our projects. Then we can only rely on `microerror`.